MekHQ/src/mekhq/campaign/unit/Unit.java

```diff
@@ -808,37 +803,25 @@ public String getDescHTML() {
     }
 
     public TargetRoll getSiteMod() {
-        switch (site) {
-            case SITE_FIELD:
-                return new TargetRoll(2, "in the field");
-            case SITE_MOBILE_BASE:
-                return new TargetRoll(1, "field workshop");
-            case SITE_BAY:
-                return new TargetRoll(0, "transport bay");
-            case SITE_FACILITY:
-                return new TargetRoll(-2, "maintenance facility");
-            case SITE_FACTORY:
-                return new TargetRoll(-4, "factory");
-            default:
-                return new TargetRoll(0, "unknown location");
-        }
+        return switch (site) {
+            case SITE_FIELD -> new TargetRoll(2, "in the field");
```

Comment:
PG 193 of CO has the new names for these. It would be nice to move to these with a seperate PR. 

<img width="660" alt="image" src="https://github.com/user-attachments/assets/bf554474-f603-4f92-9fcf-d985aa2cf269">

Reply:
I'll address that in a separate PR, but gimme a couple of minutes and I'll see it done